packages/tools/src/stateManagement/annotation/annotationState.ts

```diff
@@ -123,7 +123,7 @@ function removeAnnotation(
  * Get the Annotation object by its UID
  * @param annotationUID - The unique identifier of the annotation.
  * @param element - The element that the tool is being used on.
- * @returns A Annotation object.
+ * @returns A Annotation object or undefined if that does not exist.
```

Comment:
I think the return type is also incorrect (has happend before this PR)

Reply:
Glad that I forgot that.
I will remove this, but I would like to do an observation that the signature of it is not consistent to
annotationManager.getAnnotation(annotationUID);